src/debugger/launcher.ts

```diff
@@ -32,17 +34,22 @@ export class Launcher {
                     Log.logError("The target platform could not be read. Did you forget to add it to the launch.json configuration arguments?");
                 } else {
                     const sourcesStoragePath = path.join(this.projectRootPath, ".vscode", ".react");
-                    const packager = new Packager(this.projectRootPath, sourcesStoragePath);
                     return Q({})
                         .then(() => {
                             generator.step("startPackager");
-                            return packager.start();
+                            return extensionMessageSender.sendMessage(em.ExtensionMessage.START_PACKAGER);
+                        })
+                        .then(() => {
+                            let scriptImporter = new ScriptImporter(sourcesStoragePath);
+                            return scriptImporter.downloadDebuggerWorker(sourcesStoragePath).then(() => {
+                                Log.logMessage("Downloaded debuggerWorker.js (Logic to run the React Native app) from the Packager.");
+                            });
                         })
                         // We've seen that if we don't prewarm the bundle cache, the app fails on the first attempt to connect to the debugger logic
                         // and the user needs to Reload JS manually. We prewarm it to prevent that issue
                         .then(() => {
                             generator.step("prewarmBundleCache");
-                            return packager.prewarmBundleCache(runOptions.platform);
+                            return extensionMessageSender.sendMessage(em.ExtensionMessage.PREWARM_BUNDLE_CACHE, [runOptions.platform]);
```

Comment:
Oh, You have added in the pass through, it just isn't used here or further above yet